docker-compose.yml

```
@@ -1,6 +1,18 @@
 version: '3'
 services:
   # ARM64 dependencies
+  aws_sns_arm64:
+    image: localstack/localstack
+    environment:
+      - SERVICES=sns
+      - DEBUG=1
+      - DATA_DIR=/tmp/localstack/data
+      - DEFAULT_REGION=us-east-1
+    ports:
+      - "4566:4566"
+    volumes:
+      - "./.localstack:/tmp/localstack"
```

I think it would probably be useful to include this data as part of the CI log upload?

```suggestion
      - "./tracer/build_data/localstack:/tmp/localstack"
```